src/main/webapp/register.html

```diff
@@ -0,0 +1,100 @@
+    <script src="register.js"></script>
+  </head>
+  <body onload="prepareFormValidation()">
+    <div id="content">
+      <nav class="navbar navbar-expand-lg navbar-light bg-light">
+        <span class="navbar-brand mb-0 h1">InterviewMe</span>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" 
+          data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" 
+          aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+          <div class="navbar-nav">
+            <a class="nav-item nav-link active" href="#">Home</a>
+            <!--TODO: Put in links to other pages. Put on all pages-->
+            <a class="nav-item nav-link" href="profile.html">Profile</a>
+            <a class="nav-item nav-link" href="scheduled-interviews.html">Events</a>
+            <a class="nav-item nav-link" href="availability.html">Availability</a>
+            <a class="nav-item nav-link" href="#">Scheduling</a>
+            <!--TODO: Move this to the right of the navbar and allow user to log in and out using Authentication-->
+            <a class="nav-item nav-link" href="#">Logout</a>
+          </div>
+        </div>
+      </nav>
+      <br>
+      <h1 class="text-center">Register for InterviewMe</h1>
+      <h5 class="text-center lead">Fill in the following fields below.</h5>
+      <br>   
+      <!-- TODO: copy to profile page once this is approved so that the forms match. -->
+      <form class="needs-validation" novalidate action="/register" method="POST">
+        <div class="form-row">
+          <div class="col-md-12 mb-3">
+            <label for="user-email">Email</label>
+            <!-- TODO: replace with user's actual email -->
+            <input type="text" readonly class="form-control" id="user-email" value="tess@gmail.com">
+          </div>
+        </div>
+        <div class="form-row">
+          <div class="col-md-6 mb-3">
+            <label for="first-name-field">First name</label>
+            <input type="text" class="form-control" id="first-name-field" placeholder="First name" value="" required>
+            <div class="valid-feedback">
+              Looks good!
+            </div>
+          </div>
+          <div class="col-md-6 mb-3">
+            <label for="first-name-field">Last name</label>
+            <input type="text" class="form-control" id="last-name-field" placeholder="Last name" value="" required>
+            <div class="valid-feedback">
+              Looks good!
+            </div>
+          </div>
+        </div>
+        <div class="form-row">
+          <div class="col-md-6 mb-3">
+            <label for="company-field">Company</label>
+            <input type="text" class="form-control" id="company-field" placeholder="Google" value="">
+          </div>
```

Comment:
Is there a reason this one didn't get a check mark?

Reply:
Seems like this only happens when you click a suggestion. The background image (check mark) doesn't show. But when you type it in, it's fine. l'll update the screenshot.